GL_Framework/libs/assertpy_extensions.py

```diff
@@ -0,0 +1,29 @@
+import os
+from typing import Optional
+import jsonschema
+from assertpy import add_extension
+from json import loads
+from constants import SCHEMAS_DIRECTORY
+import allure
+
+
+@allure.step("Validate json schema")
+def schema_valid(self, schema_name: Optional[str], status_code: int = 200):
+    if self.val.status_code != status_code:
```

Comment:
The response status code verification should be moved to the API client or session class, and expected status code should not be hardcoded